install.sh

```diff
@@ -1161,15 +1179,43 @@ set_githooks_directory() {
         git config --global githooks.useCoreHooksPath yes
         git config --global githooks.pathForUseCoreHooksPath "$1"
         git config --global core.hooksPath "$1"
+
+        CURRENT_TEMPLATE_DIR=$(git config --global init.templateDir)
+
+        # shellcheck disable=SC2012
+        if [ -d "$CURRENT_TEMPLATE_DIR" ] &&
+            [ "$(ls -1 "$CURRENT_TEMPLATE_DIR/hooks" 2>/dev/null | wc -l)" != "0" ]; then
+            echo "! The \`init.templateDir\` setting is currently set to" >&2
+            echo "  \`$CURRENT_TEMPLATE_DIR\`" >&2
+            HOOKS_GET_IGNORED=1
+        fi
+
+        # shellcheck disable=SC2012
+        if [ -d "$GIT_TEMPLATE_DIR" ] &&
+            [ "$(ls -1 "$GIT_TEMPLATE_DIR/hooks" 2>/dev/null | wc -l)" != "0" ]; then
+            echo "! The environment variable \`GIT_TEMPLATE_DIR\` is currently set to" >&2
+            echo "  \`$GIT_TEMPLATE_DIR\`" >&2
+            HOOKS_GET_IGNORED=1
+        fi
+
+        if [ "$HOOKS_GET_IGNORED" = 1 ]; then
```

Comment:
Nice one!
Some minor edit suggestion perhaps:
```
            echo "  and it contains Git hooks which get installed," >&2
            echo "  but will be ignored because \`core.hooksPath\` is also set." >&2
            echo "  It is recommended to either remove the files, or run the Githooks" >&2
            echo "  installation without the \`--use-core-hookspath\` parameter." >&2
            unset HOOKS_GET_IGNORED
```

We're not failing the install here, right? In which case the above changes should soften the message to a recommendation, instead of sounding like something you have to do.

Reply:
Ups should be = "1"

Reply:
Jeah I thought so too, will change that